joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -181,6 +186,9 @@ controller_interface::return_type JointTrajectoryController::update(
     if (!(*traj_point_active_ptr_)->is_sampled_already())
```

Comment:
Since `*traj_point_active_ptr_` is used a lot and it's kind of annoying to read, you could do something like this:

`auto active_traj_point = *traj_point_active_ptr_;`